runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsPlugin.java

```diff
@@ -159,11 +169,15 @@ public void onClickMenu(final MenuOptionClicked event)
 
 		if (event.getMenuOption().equals(MENU_SET))
 		{
-			tags.put(event.getId(), selectedMenu);
+			setTag(event.getId(), selectedMenu);
+
+			checkForTags(client.getItemContainer(InventoryID.INVENTORY));
```

Comment:
Is this really necessary? Obviously we have tag here, so just set the bool to true.